### ISSUE TITLE
automatic import of flowcheck/assert module

### ISSUE THREAD

**Issue by gcanti** [gcanti/flowcheck#4]

**Comment by gcanti** [comment]
My plan:

- for now automatically handle only the commonjs case
- change the default namespace from `f` to `_f` to avoid likely name conflicts
- add an `importVisitor` to `visitor.js`. It adds `var _f = require("flowcheck/assert");` at the beginning of the file
- add a boolean option `skipImport` (default `false`). If set to `true`, skips the `importVisitor` (useful for tests or if you want to import by hand the module flowcheck/assert as a global)
- add to importVisitor a `namespace.indexOf('require') === -1` check in order to not break flowcheck-loader (temporary)

example:

```js
var n: number = 1;
```

will translate to 

```js
var _f = require("flowtype/assert");

var n: number = _f.check(1, _f.number);
```

cc @gaearon

**Comment by gaearon** [comment]
Sounds sensible. Let me know when you release & I'll update the loader.

**Comment by gcanti** [comment]
released v0.2.1
